### ISSUE TITLE
Paper reproduction 

### ISSUE THREAD

**Issue by 15383269852** [chufengt/iccv19_attribute#67]
Hello, I reproduced very poorly on the RAP dataset, and the difference was 6% on Recall. Do you have any suggestions for this? The environment I use is the same as mentioned in the paper

**Comment by chufengt** [comment]
Sorry for the late reply. 

1. Have you tried to reproduce the results with the provided checkpoint? It can help you debug problems beside training.
2. Have you tried the warm-up trick mentioned in #7?

**Comment by 15383269852** [comment]
spcifically, what is the exact hyper-parameters config of rap?i tried to reproduce the results with the provided checkpoin,but the effect is very poor

**Comment by 15383269852** [comment]
spcifically, what is the exact hyper-parameters config of rap? I tried to reproduce the results with the provided checkpoint,but the effect is very poor

**Comment by 15383269852** [comment]
[
![21](https://user-images.githubusercontent.com/92853114/163297546-3c640ab6-e833-478c-a69a-0edca43eed1f.jpg)
](url)

**Comment by suyashhchougule** [comment]
> [ ![21](https://user-images.githubusercontent.com/92853114/163297546-3c640ab6-e833-478c-a69a-0edca43eed1f.jpg) ](url)

were you able to reproduce the results finally ?
